deployment/CP05MOAS-A6264_Deploy.csv

```diff
@@ -1,99 +1,99 @@
 WJ20190709,,WJ20190709,,CP05MOAS-A6264-04-NUTNRN000,13,1,2019-07-11T13:50:00,2019-07-11T23:30:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-NUTNRN-00367,40.33785,-70.904967,,,,custom bottom boundary layer mission on across-shelf line
 WJ20190709,,WJ20190709,,CP05MOAS-A6264-05-ADCPAN000,13,1,2019-07-11T13:50:00,2019-07-11T23:30:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-ADCPAN-22077,40.33785,-70.904967,,,,custom bottom boundary layer mission on across-shelf line
 WJ20190709,,WJ20190709,,CP05MOAS-A6264-06-PARADN000,13,1,2019-07-11T13:50:00,2019-07-11T23:30:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-PARADN-50180,40.33785,-70.904967,,,,custom bottom boundary layer mission on across-shelf line
-AR-39,,AR-39,,CP05MOAS-A6264-00-ENG000000,14,1,2019-09-30T12:03:00,2019-09-30T21:59:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGCON-AUVPNR-06264,39.92068,-70.764267,,,,abbreviated mission due to time constraints (AC-4 to AC-1)
-AR-39,,AR-39,,CP05MOAS-A6264-01-FLORTN000,14,1,2019-09-30T12:03:00,2019-09-30T21:59:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-FLORTN-01109,39.92068,-70.764267,,,,abbreviated mission due to time constraints (AC-4 to AC-1)
-AR-39,,AR-39,,CP05MOAS-A6264-02-DOSTAN000,14,1,2019-09-30T12:03:00,2019-09-30T21:59:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-DOSTAN-01520,39.92068,-70.764267,,,,abbreviated mission due to time constraints (AC-4 to AC-1)
-AR-39,,AR-39,,CP05MOAS-A6264-03-CTDAVN000,14,1,2019-09-30T12:03:00,2019-09-30T21:59:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-CTDAVN-00425,39.92068,-70.764267,,,,abbreviated mission due to time constraints (AC-4 to AC-1)
-AR-39,,AR-39,,CP05MOAS-A6264-04-NUTNRN000,14,1,2019-09-30T12:03:00,2019-09-30T21:59:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-NUTNRN-00367,39.92068,-70.764267,,,,abbreviated mission due to time constraints (AC-4 to AC-1)
-AR-39,,AR-39,,CP05MOAS-A6264-05-ADCPAN000,14,1,2019-09-30T12:03:00,2019-09-30T21:59:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-ADCPAN-22077,39.92068,-70.764267,,,,abbreviated mission due to time constraints (AC-4 to AC-1)
-AR-39,,AR-39,,CP05MOAS-A6264-06-PARADN000,14,1,2019-09-30T12:03:00,2019-09-30T21:59:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-PARADN-50180,39.92068,-70.764267,,,,abbreviated mission due to time constraints (AC-4 to AC-1)
+AR39,,AR39,,CP05MOAS-A6264-00-ENG000000,14,1,2019-09-30T12:03:00,2019-09-30T21:59:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGCON-AUVPNR-06264,39.92068,-70.764267,,,,abbreviated mission due to time constraints (AC-4 to AC-1)
+AR39,,AR39,,CP05MOAS-A6264-01-FLORTN000,14,1,2019-09-30T12:03:00,2019-09-30T21:59:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-FLORTN-01109,39.92068,-70.764267,,,,abbreviated mission due to time constraints (AC-4 to AC-1)
+AR39,,AR39,,CP05MOAS-A6264-02-DOSTAN000,14,1,2019-09-30T12:03:00,2019-09-30T21:59:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-DOSTAN-01519,39.92068,-70.764267,,,,abbreviated mission due to time constraints (AC-4 to AC-1)

```

Comment:
Looks good. Associated checkout document .zip file will need to be revised - remove the 1520 cal doc and insert the 1519 cal doc. (should be 3403-40001-00022-A.zip but zip file does not exist in Vault, just the associated checkout test)